docs/section5/Rule5-39.md

```diff
@@ -1,32 +1,34 @@
-
 # Envelope - Rule 5-39  
-
+**Schema Version** 0.0.29  
+**Primary Rule:** False  
 **Rule ID:** 5-39  
-**Rule Description:** It is acceptable to use either an annual average ground temperature or monthly average ground temperatures for calculation of heat loss through basement floors.  
-**Rule Assertion:** B-RMR ASHRAE229:weather.monthly_ground_temperature != NULL  
+**Rule Description:** It is acceptable to use either an annual average ground temperature or monthly average ground temperatures for calculation of heat loss through below-grade walls and basement floors.  
 **Appendix G Section:** Section 5 Envelope  
 **Appendix G Section Reference:** Section G3.1-14(b) Building Envelope Modeling Requirements for the Proposed design and Baseline  
 
-**Applicability:** All required data elements exist for B_RMR  
-**Applicability Checks:**  None  
-
-**Manual Check:** Yes  
-**Evaluation Context:** Each Data Element  
+**Applicability:** All required data elements exist for B_RMD  
+**Applicability Checks:**  
+  1. B_RMD contains at least 1 basement floor or below grade wall surface 
+ 
+**Evaluation Context:** RMD  
 **Data Lookup:** None  
-**Function Call:** None  
+**Function Calls:**
+get_opaque_surface_type()
 
 ## Rule Logic:  
+**Applicability Check 1**  
+- If for-loop completes and no below-grade wall surfaces were found, rule is not applicable. (no below-grade walls also means no basement floors) : `is_applicable = false`  
+- For each surface in the B_RMD: `for surface in B_RMD...Surface:`  
+  - If the surface is a below-grade wall rule is applicable and continue to rule assertion: `if get_opaque_surface_type(surface) == "BELOW-GRADE WALL": is_applicable = true; break`  
 
 **Rule Assertion:**  
-
-- Case 1: If B-RMR includes annual average or monthly average ground temperature: `if B_RMR.ASHRAE229.weather.monthly_ground_temperature: PASS`
-
-- Case 2: Else: `else: FAIL`
+- Case 1: If B_RMD has no below-grade wall surfaces and therefore also no basement floor surfaces: `if not is_applicable: NOT_APPLICABLE`  
+- Case 2: If RPD includes a ground temperature schedule: `if RPD.weather.ground_temperature_schedule: UNDETERMINED`  
```

Comment:
Should we include a message to tell the review what to check? I think this is useful.

Reply:
Is this undetermined because:
1) we can't tell if the ground temperature was used to calculate heat loss? 
2) we don't know if the ground temperature used is representative of project climate?
3) both?